kedro_sagemaker/generator.py

```diff
@@ -226,6 +227,31 @@ def generate(self) -> SageMakerPipeline:
         )
         return smp
 
+    def _add_mlflow_support(self, steps, runner_config):
+        mlflow_start_run = self._create_processing_step(
+            node=SimpleNamespace(name="start-mlflow-run"),
```

Comment:
Why it is not a `node` from Kedro?

Reply:
Good question! I planned it to be `kedro.pipeline.node`, but it's not trivial to mimic one:

```
>>> from kedro.pipeline import node
>>> node(name='start-mlflow-run')
TypeError: node() missing 3 required positional arguments: 'func', 'inputs', and 'outputs'
>>> node(name='start-mlflow-run', func=None, inputs=None, outputs=None)
ValueError: Invalid Node definition: first argument must be a function, not 'NoneType'.
Format should be: node(function, inputs, outputs)
>>> node(name='start-mlflow-run', func=lambda x: x, inputs=None, outputs=None)
ValueError: Invalid Node definition: it must have some 'inputs' or 'outputs'.
Format should be: node(function, inputs, outputs)
```

And since the generator only uses the `node.name` for processing steps, I decided to go the fake-node approach

Reply:
Hacky, but ok :P